client/src/Interfaces/IUser.ts

```diff
@@ -0,0 +1,8 @@
+interface User {
+    name : string,
+    email? :string,
+    password? :string,
```

Comment:
also i have interface User, we can use one of them

Reply:
I'll fix this once either one is merged 🙏🏼